controllers/postArticlesController.js

```diff
@@ -0,0 +1,40 @@
+const postComment = require("../models/postArticlesModel");
+
+async function postArticleComments(req, res, next) {
+  try {
+    if (!req.body.username || !req.body.body) {
+      return res.status(400).send({ error: "No username or body found" });
+    }
+
+    const { article_id } = req.params;
+    const { username, body } = req.body;
+
+    if (isNaN(article_id)) {
+      return res.status(400).send({ message: "Invalid input" });
+    }
+    const userComment = await postComment(article_id, username, body);
+    if (!userComment ) {
+        res.status(404).send({
+          message: "Comment not found",
+        });
+      }
```

Comment:
This would be better handled in the model.
Im also not sure what the error message means, as they're posting the comment. Should it be article id not found?